cypress/e2e/Cart.ts

```diff
@@ -0,0 +1,40 @@
+import { Given, When, Then,  After } from "@badeball/cypress-cucumber-preprocessor"
+
+Given('I visit the page', ()=>{
+    cy.visit('https://demo.nopcommerce.com/')
+})
+
+When('Redirect to the Notebooks PLP',()=>{
+    cy.get('li').contains('Notebooks ').click({ force: true })
+    cy.get('.page-title').should('have.text', 'Notebooks')
+    
+})
+
+Then('Redirect to the Notebooks PDP',()=>{
+    cy.get('.product-item').first().click()
+})
+
+Then('Click the add to cart button', ()=>{
+    cy.get('#add-to-cart-button-4').click()
+    cy.wait(4000)
```

Comment:
Why such a long wait? I don't think it's needed.
Maybe try using cypress-wait-until plugin

Reply:
I have left this wait, thinking it was the reason, but it's not necessary because sometimes the page has a bug, and that message does not appear

So it's working without wait